Example/package.json

```diff
@@ -27,22 +27,24 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.28",
     "@fortawesome/free-solid-svg-icons": "^5.13.0",
     "@fortawesome/react-native-fontawesome": "^0.2.3",
-    "@react-native-community/masked-view": "^0.1.10",
     "@react-native-community/picker": "^1.8.1",
     "@react-native-community/slider": "^4.4.0",
-    "@react-navigation/native": "^5.9.2",
-    "@react-navigation/stack": "^5.14.2",
+    "@react-native-masked-view/masked-view": "^0.2.8",
+    "@react-navigation/native": "^6.0.12",
+    "@react-navigation/native-stack": "^6.9.1",
+    "@react-navigation/stack": "^6.2.3",
     "@react-navigation/web": "^1.0.0-alpha.9",
     "async-retry": "^1.3.1",
     "d3-shape": "^1.3.7",
     "expo-asset": "^8.2.0",
+    "patch-package": "^6.4.7",
     "react": "18.2.0",
     "react-dom": "^16.13.1",
     "react-native": "0.71.0-rc.5",
     "react-native-gesture-handler": "^2.8.0",
     "react-native-pager-view": "^5.4.1",
     "react-native-safe-area-context": "^4.4.1",
-    "react-native-screens": "^3.18.2",
+    "react-native-screens": "software-mansion/react-native-screens#@piaskowyk/shared-transition-element-v2",
```

Comment:
I think I just copied the deps from the SET branch, since I wanted to test on a newer `navigation` version.
This would need to be reverted to a normal `screens` version, along with the `navigation` packages' versions as well, before merging.